src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { ChangeEvent, FormEvent, useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  onAddMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ onAddMovie }) => {
+  const [searchValue, setSearchValue] = useState('');
+  const [foundMovie, setFoundMovie] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleFindMovie = (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    setIsLoading(true);
+
+    getMovie(searchValue)
+      .then((movieData) => {
```

Comment:
Let’s use `async await` all over the code. It gives you a simpler way to work with asynchronous promise-based code